src/workingtitle-vcockpits-instruments-cj4/html_ui/Pages/VCockpit/Instruments/Airliners/CJ4/MFD/CJ4_MFD.js

```diff
@@ -210,6 +210,14 @@ class CJ4_MFD extends BaseAirliners {
                 this.systemOverlay.show(false);
             }
 
+            const rangeSelectDisabled = WTDataStore.get('WT_CJ4_RANGE_SEL_DISABLED', 0);
```

Comment:
Substantive change here. Default for the range sel is now enabled, and the code we had in the element itself is pulled up here.